packages/language-support/src/tests/autocompletion/databasenameCompletion.test.ts

```diff
@@ -15,6 +27,32 @@ describe('Can complete database names', () => {
     },
   };
 
+  test('Correctly completes database names/aliases with special symbols using backticks', () => {
+    const query = 'SHOW DATABASE ';
+
+    testCompletions({
+      query,
+      dbSchema,
+      expected: [
+        {
+          label: 'financial reports',
+          kind: CompletionItemKind.Value,
+          insertText: '`financial reports`',
+        },
+        {
+          label: 'panama papers1',
+          kind: CompletionItemKind.Value,
+          insertText: '`panama papers1`',
+        },
+        {
+          label: 'office-db',
+          kind: CompletionItemKind.Value,
+          insertText: '`office-db`',
+        },
+      ],
+    });
+  });
+
   test('Correctly completes database names and aliases in SHOW DATABASE', () => {
```

Comment:
There's also a `SHOW ALIAS ` test in this file which you should add the corresponding checks to